get-started/index.md

```diff
@@ -44,36 +44,44 @@ $ kubectl get pods -n olm
   summary="Cryostat on OperatorHub"
   image-name="cryostat-operatorhub-search.png"
 %}
-Use the search bar to find the Cryostat (provided by Red Hat) catalog item.
+Use the search bar to find the **Red Hat build of Cryostat** catalog item.
 {% include howto_step.html
   summary="Select the Cryostat Operator and click the Install button"
   image-name="cryostat-operatorhub-install.png"
 %}
-Choose the namespace for Cryostat to be deployed into. This should be the same namespace that contains your JVM applications which you intend to monitor or profile using Cryostat.
+Choose your Operator installation mode:
+1. In **All Namespaces** installation mode, the Cryostat Operator instance will watch for `Cryostat` or
+`ClusterCryostat` Custom Resources (`CR`s) created in any Namespace and create corresponding Cryostat instances.
+2. In the **A specific namespace** installation mode, you must also select an installation Namespace, and the Cryostat
+Operator instance will only watch for `Cryostat` or `ClusterCryostat` instances created in that same Namespace.
 {% include howto_step.html
   summary="Install the Operator"
   image-name="cryostat-operatorhub-install-in-progress.png"
 %}
-Click "Install" and wait for the installation to complete.
+Click "Install" and wait for the installation to complete. In this example we will proceed with **All Namespaces**.
 {% include howto_step.html
   summary="Create a Cryostat instance"
   image-name="cryostat-operatorhub-install-complete.png"
 %}
-Once the installation is complete, click "Create Cryostat" to create a Cryostat Custom Resource instance. This provides configuration information for the Operator to know
-the specifics of how to deploy your Cryostat instance. Continue to [Setup](#setup).
+Once the installation is complete, click **Create Cryostat** to create a `Cryostat` `CR` instance. This provides
+configuration information for the Operator to know the specifics of how to deploy your Cryostat instance.
+Continue to [Setup](#setup).
+
+**Note:** Alternative methods for installing the Operator or using the  `ClusterCryostat` `CR` are described in
```

Comment:
Not sure why, I don't see this **Note** bolded on the website.

Reply:
Oh never mind, some git issue

Reply:
Maybe restart your `jekyll` server or do a force-refresh on the website (bypass cache)?

![image](https://github.com/cryostatio/cryostatio.github.io/assets/3787464/4edf1f0b-8d23-4d22-8f5b-60d9fa2150c7)